src/Button/ZoomToExtentButton/ZoomToExtentButton.tsx

```diff
@@ -93,6 +110,9 @@ class ZoomToExtentButton extends React.Component<ZoomToExtentButtonProps> {
     if (!view) { // no view, no zooming
       return;
     }
+    if (!extent && (!center && !zoom)) {
+      logger.warn('Provide either an extent or a center and a zoom.');
+    }
```

Comment:
please add a return statement in the if-block, as the code shall not continue without valid parameters